sys/netinet/cc/cc_cdg.c

```diff
@@ -308,7 +309,7 @@ cdg_cb_init(struct cc_var *ccv, void *ptr)
 	cdg_data->queue_state = CDG_Q_UNKNOWN;
 	cdg_data->maxrtt_in_rtt = 0;
 	cdg_data->maxrtt_in_prevrtt = 0;
-	cdg_data->minrtt_in_rtt = INT_MAX;
+	cdg_data->minrtt_in_rtt = UINT_MAX;
```

Comment:
minrtt_in_rtt is just int, this change makes no sense to me, same as one below.

Reply:
Oops! I am not sure why I did that. Good catch.